client/src/features/sessionsV2/StartSessionButton.tsx

```diff
@@ -50,6 +50,7 @@ export default function StartSessionButton({
         "px-2"
       )}
       to={startUrl}
+      data-cy="session-btn"
```

Comment:
I would add `start` to avoid it being too generic.
Also, you used `-button` in most other properties except the sessions components; let's keep it consistent.

```suggestion
      data-cy="start-session-button"
```